### PR TITLE
{lib,vis}[foss/2022a] devbio-napari v0.10.1, PyOpenCL v2023.1.4, autopep8 v2.0.4, ...

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/d/devbio-napari/devbio-napari-0.10.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    ('napari-plot-profile', '0.2.2', {
+        'checksums': ['1a1f087b23eaa71a16f0ffde66e965fd7f6b69e5d52b3664c7db295f30fed501'],
+    }),
+    ('napari-plugin-search', '0.1.4', {
+        'checksums': ['6dcc0793d072777c8d0cfebff89281252e57e13f8e13a8e0458a9ba7d86a06a0'],
+    }),
+    ('napari_pyclesperanto_assistant', '0.22.1', {
+        # import napari_pyclesperanto_assistant fails without GUI
+        # but it has been tested with GUI with following command:
+        # `unset SESSION_MANAGER && python -c "import napari_pyclesperanto_assistant"`
```
Comment (boegel) [inline easybuild/easyconfigs/d/devbio-napari/devbio-napari-0.10.1-foss-2022a.eb:172]: We should mention here why the `unset SESSION_MANAGER` is required:

```
WARNING: Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3157.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/6757166fb396896283e00c4c33d37a2c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**